spec/graphql/errors_spec.rb

```diff
@@ -12,7 +12,25 @@
   end
 
   describe '#rescue_from' do
-    it 'raises an exception if the object which was passed is not a class' do
+    before do
+      module NetworkErrors
```

Comment:
we could potentially define it outside the `before` block which is being triggered for every test to avoid:

> warning: already initialized constant NetworkErrors::ERRORS